go-controller/pkg/node/gateway_hybrid.go

```diff
@@ -327,6 +327,18 @@ func addDefaultConntrackRulesHybrid(nodeName, gwBridge, gwIntf string, stopChan
 
 	}
 
+	if config.IPv6Mode {
+		// REMOVEME(trozet) when https://bugzilla.kernel.org/show_bug.cgi?id=11797 is resolved
```

Comment:
@trozet I suspect this will never get fixed; upstream comment seems germane as solicitation/etc isn't a "connection" really, so why would conntrack do anything about it.

Reply:
yeah it may never be "fixed" by allowing tracking state for those different packets. Either way this would need to stay around so we can ensure both sides get the updates.